.one-pipeline-cd.yaml

```diff
@@ -23,6 +23,11 @@ setup:
       echo "${DIGEST}"
       echo "${APP}" | jq '.'
 
+      if [[ ${ARTIFACT} == *"icr.io/appcafe/open-liberty/samples/"* ]];then
```

Comment:
It would probably be best to have quotes around the '${ARTIFACT}' I think?

Reply:
Added quotes